### DISCUSSION TITLE
Homework 02

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#7] [Announcements]
Homework 02 is released and it is due on Friday February 18 cb17618

You can see the equation rendered page here: https://stat5125-uconn.github.io/2022Spring/homework/hw02/

**Comment by Xiw2000** [comment]
Hi Dr. Wang,

"Without using any additional packages" I wonder if using statistic can be used here.

**Reply by Ossifragus** [reply]
No. The problem means to ask you code the formula manually.

**Comment by jguil2022** [comment]
Where should be upload the assignment? Are you going to setup assignments like the previous HW?

**Reply by Ossifragus** [reply]
Look at cb17618
